lib/tree.rb

```diff
@@ -1,38 +1,63 @@
 class NoApplesError < StandardError; end
 
 class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+  #fill_in :height, :age, :apples, :alive
+  attr_accessor :height
+  attr_accessor :age
+  attr_accessor :apples
+  attr_accessor :alive
 
   def initialize
+    @height = 0
+    @age = 0
+    @apples = []
+    @alive = true
   end
 
   def age!
+    @age += 1
+    @height += rand(3..50)
+    if @age > 70
+      @alive =false
+    elsif @age > 5 && @alive == true
```

Comment:
maybe abstract the line 22 conditions into another method that checks if it's 'of age' for growing apples. Note: check the line 21 issue with =false (no space). Try to catch those things on future PRs (but no need to re-PR for that (for PairColumbus stuff)).